bundles/org.openhab.binding.grundfosalphareader/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="grundfos_flow">
```

Comment:
Please adapt according to [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention).
```suggestion
	<channel-type id="grundfos-flow">
```